libraries/protocol/include/scorum/protocol/config.hpp

```diff
@@ -120,14 +120,14 @@
 #define SCORUM_MAX_MEMO_SIZE                   2048
 #define SCORUM_MAX_PROXY_RECURSION_DEPTH       4
 
-#define SCORUM_VESTING_WITHDRAW_INTERVALS           13
+#define SCORUM_VESTING_WITHDRAW_INTERVALS           52
 #define SCORUM_VESTING_WITHDRAW_INTERVAL_SECONDS    (DAYS_TO_SECONDS(7)) /// 1 week per interval
```

Comment:
Why I can't receive SCR every day? ;( We have enough precision for this.
If you want to left 1 year withdraw period just use 365 and DAYS_TO_SECONDS(1).